composer.json

```diff
@@ -19,7 +19,7 @@
         "dmstr/yii2-db": "*",
         "friendsofphp/php-cs-fixer": "1.* || 2.*",
         "yiisoft/yii2": "~2.0.13",
-        "yiisoft/yii2-gii": "~2.0.6"
+        "yiisoft/yii2-gii": "2.*"
```

Comment:
This should be more like: `~2.0.6 || ~2.1.0`

Reply:
yeah, you are right.